### DISCUSSION TITLE
SMTP Help

### DISCUSSION THREAD

**Discussion by jmourinho** [Leantime/leantime#1366] [Q&A]
Hi, 

since i upgraded to this last version (from 2.2.x) i am not receiving any emails whatsoever. I had not touched the SMTP settings.

Have you made any changes to the code in what concerns to mail notifications? It's not a problem with our SMTP server as other apps are sending mails as usual.

Thanks

**Comment by marcelfolaron** [comment]
Do you see any errors in logs/error.log ? 
Is the variable `public $email;` in your config file set to valid email address?
